go/hyperkit.go

```diff
@@ -675,3 +672,13 @@ func getHome() string {
 	}
 	return os.Getenv("HOME")
 }
+
+// defaultString returns the first non empty string, or "" if there is none.
+func defaultString(ss ...string) string {
+	for _, s := range ss {
+		if s != "" {
+			return s
+		}
+	}
+	return ""
+}
```

Comment:
This is not a simplification over the existing code, please leave the code as it was.

Reply:
The function will be used elsewhere, that's not its single use.

Reply:
Then it should go into the PR which does this. It would also be good to mention it in the commit message as i proposed here https://github.com/moby/hyperkit/pull/166/commits/c202f0709cbc834972a91f58990875f1d0a34ac7#r152323406

Reply:
That's probably not the link you meant to sent, but rather https://github.com/moby/hyperkit/pull/166/commits/c202f0709cbc834972a91f58990875f1d0a34ac7.  But maybe GitHub is playing tricks on URLs here.

I'll move it into another PR.